pvlib/iotools/pvgis.py

```diff
@@ -665,3 +665,46 @@ def read_pvgis_tmy(filename, pvgis_format=None, map_variables=None):
         data = data.rename(columns=VARIABLE_MAP)
 
     return data, months_selected, inputs, meta
+
+
+def get_pvgis_horizon(latitude, longitude,
+                      url=URL, **kwargs):
+    r'''
+    Get horizon data from PVGIS
```

Comment:
```suggestion
    """Get horizon data from PVGIS.
```
Just some Python formatting